x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/details.tsx

```diff
@@ -13,38 +13,46 @@ import {
   EuiDescriptionList,
   EuiLoadingContent,
   EuiHorizontalRule,
+  EuiHealth,
   EuiSpacer,
+  EuiListGroup,
+  EuiListGroupItem,
 } from '@elastic/eui';
 import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n/react';
 import { useKibana } from '../../../../../../../../src/plugins/kibana_react/public';
-import { useManagementListSelector } from './hooks';
+import { EndpointMetadata } from '../../../../../common/types';
+import { useHostListSelector } from './hooks';
 import { urlFromQueryParams } from './url_from_query_params';
-import { uiQueryParams, detailsData, detailsError } from './../../store/managing/selectors';
+import { FormattedDateAndTime } from '../formatted_date_time';
+import { uiQueryParams, detailsData, detailsError } from './../../store/hosts/selectors';
 
-const HostDetails = memo(() => {
-  const details = useManagementListSelector(detailsData);
-  if (details === undefined) {
-    return null;
-  }
+const HostDetails = memo(({ details }: { details: EndpointMetadata }) => {
+  const HostIds = styled(EuiListGroupItem)`
+    margin-top: 0;
+    .euiListGroupItem__text {
+      padding: 0;
+    }
+  `;
```

Comment:
You should not define `HostIds` component inside of `HostDetails` - you only need to define the styled component once and then re-use it. Move it to be outside (at module level).